lib/client-assets.php

```diff
@@ -1031,28 +1025,12 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$editor_settings = apply_filters( 'block_editor_settings', $editor_settings, $post );
 
 	$init_script = <<<JS
-	( function() {
-		window._wpLoadBlockEditor = new Promise( function( resolve ) {
-			wp.domReady( function() {
-				resolve( wp.editPost.initializeEditor( 'editor', "%s", %d, %s, %s ) );
-			} );
-		} );
-
-		Object.defineProperty( window, '_wpLoadGutenbergEditor', {
-			get: function() {
-				// TODO: Hello future maintainer. In removing this deprecation,
```

Comment:
Confirmed: There are multiple instances of `@wordpress/deprecated` in the editor module which necessitate it remain a dependency:

https://github.com/WordPress/gutenberg/blob/ede325860ce76698d3915d045bb2e45d593f425c/packages/editor/src/components/global-keyboard-shortcuts/visual-editor-shortcuts.js#L14

https://github.com/WordPress/gutenberg/blob/ede325860ce76698d3915d045bb2e45d593f425c/packages/editor/src/components/rich-text/index.js#L50